pages/home/home.html

```diff
@@ -0,0 +1,197 @@
+        <p>
+          Oppia is a free and open-source online learning platform for collaboratively creating and sharing interactive lessons.
+        </p>
+        <p>
+          Oppia’s lesson builder allows educators to provide students with targeted, personalized   feedback that simulates conversation with an intelligent, virtual tutor.
+        </p>
+        <md-card-actions layout-xs="column">
+          <md-button class="oppia-foundation-black-text md-raised md-no-ink oppia-foundation-gold-border-button md-raised">
+            Visit Oppia.org
+          </md-button>
+        </md-card-actions>
+      </md-content>
+      <div layout="column" layout-align="center stretch">
+        <div layout="column" hide-md hide-sm hide-xs>
+          <div class="columns" >
+            <div class="column col-12">
+              <div class="device device-surface-book">
+                <div class="device-frame">
+                </div>
+                <div class="device-stripe"></div>
+                <div class="device-header"></div>
+                <div class="device-sensors"></div>
+                <div class="device-btns"></div>
+                <div class="device-power"></div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </md-card-content>
+  </md-card>
+
+  <md-card layout-fill class="oppia-foundation-white-background oppia-foundation-vertical-padding oppia-foundation-horizontal-padding">
+    <md-card-content layout-align= "center center" layout-gt-md="row" layout-xs="column">
+      <img ng-src="/static/images/home_impact.jpg" class="home-page-impact-image oppia-foundation-horizontal-padding" hide-xs>
+      <md-content layout="column" class="oppia-foundation-black-text oppia-foundation-white-background" layout-align="center stretch">
+        <h3>Impacting students around the world</h3>
+        <p>
+          Oppia was built to meet the unique needs of the developing world, where educational resources are scarce and methodologies are outdated. 
+        </p>
+        <p>
+          Our storytelling-based approach to learning requires less bandwidth than educational videos, making it accessible in areas with limited internet connections. Adaptive lessons with personalized feedback create a safe, interactive learning environment otherwise unavailable to underserved communities.
+        </p> 
+        <p>
+          By uniting a global community of volunteers and educators, Oppia’s lessons are continually translated into new languages and adapted for different cultures.
+        </p>
+      </md-content>
+    </md-card-content>
+  </md-card>
+
+  <md-card layout-fill class="oppia-foundation-green-background oppia-foundation-vertical-padding">
+    <md-card-content layout="row" layout-align="center">
+      <md-content layout="column" class="oppia-foundation-white-text oppia-foundation-green-background home-page-history-text-box">
+          <center>
```

Comment:
Use 2-space indentation

Reply:
Done.And ditto below. I think I did a lot of copy-and-pasting thus messing up the indentations..I should be more careful!